client/ayon_harmony/api/lib.py

```diff
@@ -199,17 +199,25 @@ def launch(application_path, *args):
     setup_startup_scripts()
     check_libs()
 
-    if not os.environ.get("AYON_HARMONY_WORKFILES_ON_LAUNCH", False):
+    # Whether to show workfiles on launch.
+    env_workfiles_on_launch = os.getenv( "AYON_AFTEREFFECTS_WORKFILES_ON_LAUNCH",
+        # Backwards compatibility
+        os.getenv("AVALON_AFTEREFFECTS_WORKFILES_ON_LAUNCH", True)
+    )
+    if env_value_to_bool(value=env_workfiles_on_launch):
+        ProcessContext.workfile_tool = host_tools.get_tool_by_name("workfiles")
+        host_tools.show_workfiles(save=False)
+        ProcessContext.execute_in_main_thread(check_workfiles_tool)
+
+    # If no workfile is provided, open an empty workfile.
+    if len(args) == 0:
```

Comment:
What if someone would set an additional argument in application settings - then this wouldn't apply anymore?
I feel like how we're detecting here whether a filepath is passed along is a bit off (also the existing code below). Not critical, but does smell fishy.